install-testplan-ui

```diff
@@ -92,9 +102,18 @@ def install_and_build_ui(path=TESTPLAN_UI_DIR, dev=False, verbose=False):
             production_cmd = ""
 
         logging.info("Installing Testplan UI dependencies...")
-        logging.info("Installing to path: {}".format(os.path.abspath(path)))
+        logging.info(f"Installing to path: {os.path.abspath(path)}")
+        subprocess.check_call(
+            "pnpm install{}".format(production_cmd),
+            shell=True,
+            cwd=path,
+            stdout=output,
+            stderr=subprocess.STDOUT,
+        )
+
+        logging.info("Enabling pre and post scripts...")
         subprocess.check_call(
-            "yarn install {} --network-timeout 600000".format(production_cmd),
+            "pnpm config set enable-pre-post-scripts true",
```

Comment:
PNPM does not enable pre and post scripts by default and the local config needs to be updated.

Reply:
I guess this command would fail on pnpm5?